js/scripts.js

```diff
@@ -59,24 +69,27 @@ function initMap() {
     });
 
     Promise.all([
-        fetchDataDocuments(),
+        fetchYearDocuments(),
         fetchInstitutionData(),
         // Student data is fetched again in displayMap, so if the sheet ID
-        // wasn't already in the students Map, the one from the dataDocument is used
+        // wasn't already in the students Map, the one from the yearDocument is used
         fetchStudentData(currentYear)
     ]).then(() => displayMap(currentYear, true));
 }
 
-async function fetchDataDocuments() {
-    const tabletop = await fetchTabletopData(dataDocumentsSheet);
+async function fetchYearDocuments() {
```

Comment:
Could you explain what a "year document" means?

Reply:
Yeah, I may rename this to Year Data in the future, as that's what the spreadsheet is called, but it's just the documents that pertain to a specific year of the senior map. E.x. 2019 maps to that year's sheet ID and form URL.